src/Microsoft.AspNetCore.Mvc.Razor.Precompilation.Design/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using Microsoft.AspNetCore.Mvc.Razor.Precompilation.Design.Internal;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Precompilation.Design
+{
+    public class Program
+    {
+        private readonly static Type ProgramType = typeof(Program);
+
+        public static int Main(string[] args)
+        {
+#if DEBUG
+            DebugHelper.HandleDebugSwitch(ref args);
```

Comment:
I don't want to define the type at all in release builds. AFAIK, you can't stick that attribute on a type.